IUFRO-HQ.csl

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" et-al-min="3" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>International Union of Forest Research Organizations - Headquarters</title>
+    <title-short>IUFRO - HQ</title-short>
+    <id>http://www.zotero.org/styles/international-union-of-forest-research-organizations-headquarters</id>
+    <link href="http://www.zotero.org/styles/international-union-of-forest-research-organizations-headquarters" rel="self"/>
+    <author>
+      <name>Nelson Grima</name>
+      <email>grima@iufro.org</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <updated>2020-11-23T07:51:25+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" et-al-min="3"/>
+    </names>
+  </macro>
+  <macro name="author-count">
+    <names variable="author">
+      <name form="count"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="issued">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher"/>
+      <text variable="publisher-place" suffix="."/>
+    </group>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=".">
+      <label form="short" text-case="capitalize-first" prefix="(" suffix=") "/>
+      <name and="text" et-al-min="5" initialize-with="."/>
```

Comment:
please always use `et-al-min` and `et-al-use-first` jointly to avoid confusion. Inheriting one from a higher level is super hard to follow in a style.

Reply:
Following this comment, I added 'et-al-use-first' in all instances where I used 'et-al-min'